### PR TITLE
P2 190 fe harvest log form validations

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
ticket: https://team-voyagers.atlassian.net/jira/software/projects/P2/boards/2?assignee=712020%3Afab2d613-629c-4766-961f-4fc575d34b06&selectedIssue=P2-190

Update the modules (npm install) to test.

### CODE REVIEW

File: src/components/harvestLogs/HarvestLogDrawer.tsx [hunk 1]
```diff
@@ -219,17 +221,19 @@ const HarvestLogDrawer = ({
     hideEdit();
   };
 
-  // const showEdit = () => setShowEditForm(true);
   const hideEdit = () => setShowEditForm(false);
 
+  console.log(errors);
   const onSubmit = (data: IHarvestLogForm) => {
+    console.log(errors);
```
Comment (whamester) [inline src/components/harvestLogs/HarvestLogDrawer.tsx:228]: remove console.log

File: src/components/harvestLogs/HarvestLogDrawer.tsx [hunk 2]
```diff
@@ -322,32 +338,50 @@ const HarvestLogDrawer = ({
...
-        name="seasonDeductionsIds"
-        render={({ field }) => {
+        name="seasonDeductionIds"
+        render={() => {
           return (
             <Box display="flex" flexDirection="column" gap={1}>
               <InputLabel id="harvest-log-deductions">Deduction</InputLabel>
               <Select
-                {...field}
+                // {...field}
```
Comment (whamester) [inline src/components/harvestLogs/HarvestLogDrawer.tsx:383]: remove this comments

**Review by whamester — APPROVED** [review]

File: src/components/harvestLogs/HarvestLogDrawer.tsx [hunk 3]
```diff
@@ -83,50 +85,54 @@ const HarvestLogDrawer = ({
...
+  const createHarvestLogDataList = React.useCallback(() => {
     return [
-      ["Date", formatDate(harvestLogData.createdAt || "")],
-      ["Harvest Season", harvestLogData.season?.name || ""],
-      ["Product", harvestLogData.season?.product?.name || ""],
-      ["Price Per Unit", harvestLogData.season?.price || ""],
-      ["Amount", harvestLogData.collectedAmount || ""],
-      ["Deduction", harvestLogData.totalDeduction || ""],
-      ["Note", harvestLogData.notes || ""],
+      ["Date", formatDate(Number(harvestLog?.createdAt || 0))],
```
[the remark this answers is not in the document]
Comment (ReinhardtBotha) [reply src/components/harvestLogs/HarvestLogDrawer.tsx]: https://github.com/LangaraVoyagers/project-2/pull/22/commits/37bb5e2b705b815a472430f72fd13585aadc1c33

File: src/components/harvestLogs/HarvestLogDrawer.tsx (hunk 1 above)
Comment (ReinhardtBotha) [reply src/components/harvestLogs/HarvestLogDrawer.tsx]: https://github.com/LangaraVoyagers/project-2/pull/22/commits/37bb5e2b705b815a472430f72fd13585aadc1c33

File: src/components/harvestLogs/HarvestLogDrawer.tsx (hunk 2 above)
Comment (ReinhardtBotha) [reply src/components/harvestLogs/HarvestLogDrawer.tsx]: https://github.com/LangaraVoyagers/project-2/pull/22/commits/37bb5e2b705b815a472430f72fd13585aadc1c33

**Review by ALFAROSO — APPROVED** [review]

**Review by BryanCaldeira — APPROVED** [review]

**Pull request merged**